source/_patterns/01-molecules/30-loans/11-loan-image-footer.mustache

```diff
@@ -13,6 +13,6 @@
 		<a kvtrackevent="Lending|click-Favorite star|Favorite|undefined|true" {{^is_touch}}data-tooltip aria-haspopup="true"{{/is_touch}} title="Star this loan to find it quickly later. You can view and manage your starred loans in the ‘Lend’ tab above." href="#" class="{{^is_touch}}has-tip tip-left{{/is_touch}} favorite-link{{#is_favorite}} is-favorite{{/is_favorite}}">
 	{{/is_loan_card_view}}
 
-		<svg class="icon icon-star"><use xlink:href="#icon-star"/></svg>
+		<svg class="icon icon-bookmark"><use xlink:href="#icon-bookmark"/></svg>
```

Comment:
do you need to swap between icons using the is_favorite param here too?

Reply:
I dug into this further and it's only present on the old borrower profile page. I was going to talk to you about either reverting my changes to this file, or figuring out an appropriate level to clean up these files. Gonna pick up this on Tuesday, it's pretty close to done though. I'll add screen shots and get this out of draft state on money and follow up about the cleanup.

Reply:
@mcstover I reverted the changes to this file, since it's on the  old borrower profile, which is no longer served.